golden_response/staging/get_stat_all/result.json

```diff
@@ -66,6 +66,7 @@
      ]
     },
     "Count_Person": {
+     "isDcAggregate": true,
```

Comment:
@tjann should confirm. I'm surprised why this is an aggregate.

Reply:
yeah this is surprising

Reply:
Maybe bc this is for all sources, and CensusACS5yrSurvey we aggregate for country IIRC